doc/userguide/rules/header-keywords.rst

```diff
@@ -142,19 +142,23 @@ API of MaxMind.
 
 The syntax of geoip::
 
-  geoip: src, RU;
-  geoip: both, CN, RU;
-  geoip: dst, CN, RU, IR;
-  geoip: both, US, CA, UK;
-  geoip: any, CN, IR;
+  geoip: src,RU;
+  geoip: both,CN,RU;
+  geoip: dst,CN,RU,IR;
+  geoip: both,US,CA,UK;
+  geoip: any,CN,IR;
 
 So, you can see you can use the following to make clear on which
-direction you would like to match::
+direction you would like to match
 
-  both: both directions have to match with the given geoip (geopip's)
-  any: one of the directions have to match with the given geoip ('s).
-  dest: if the destination matches with the given geoip.
-  src: the source matches with the given geoip.
+====== =============================================================
+Option Description
+====== =============================================================
+both   Both directions have to match with the given geoip (geopip's)
+any    One of the directions have to match with the given geoip ('s).
```

Comment:
nit: s/have/has/

Reply:
Hey, this is not my text ;) Fixing it :)